sagemaker-featurestore/feature_store_introduction.ipynb

```diff
@@ -310,6 +310,63 @@
     "check_feature_group_status(orders_feature_group)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add features to FeatureGroup\n",
+    "\n",
+    "After the FeatureGroup has been created, we can update the FeatureGroup by using the `UpdateFeatureGroup` API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.feature_store.feature_definition import (\n",
+    "    FractionalFeatureDefinition,\n",
+    "    IntegralFeatureDefinition,\n",
+    "    StringFeatureDefinition,\n",
+    "    FeatureTypeEnum,\n",
```

Comment:
Do we need all types of FeatureDefinition in import?

Reply:
Not required for all types here, will revise to only import StringFeatureDefinition.